rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/Diagnostics.java

```diff
@@ -52,11 +54,18 @@
 import io.usethesource.vallang.ISourceLocation;
 import io.usethesource.vallang.IString;
 import io.usethesource.vallang.IValue;
+import io.usethesource.vallang.IValueFactory;
 
 public class Diagnostics {
     private static final Logger logger = LogManager.getLogger(Diagnostics.class);
     private static final Map<String, DiagnosticSeverity> severityMap;
 
+    // Note: DiagnosticSeverity.Hint only highlightes a single character!
+    static DiagnosticSeverity errorLocationHighlight = DiagnosticSeverity.Error;
```

Comment:
are they meant to be package private and writable (also by other classes)? and why are some null? not to mention I don't like such a global state, what if we want to have it different for the rascal parser vs the DSL parsers?